jump-core/src/main/java/com/bitdecay/jump/level/builder/LevelBuilder.java

```diff
@@ -274,11 +284,11 @@ public void updateOwnNeighbors(int x, int y) {
 		}
 		// check left
 		if (ArrayUtilities.onGrid(grid, x - 1, y) && grid[x - 1][y] != null) {
-			if (grid[x-1][y].oneway) {
-				grid[x][y].collideNValue &= Direction.NOT_LEFT;
-			} else {
+			//if (grid[x-1][y].oneway) {
+			//	grid[x][y].collideNValue &= Direction.NOT_LEFT;
+			//} else {
```

Comment:
I think we still want these. Commenting these out will result in bugs when one-way platforms butt up against regular terrain.